lookout/style/format/analyzer.py

```diff
@@ -101,6 +101,7 @@ class FormatAnalyzer(Analyzer):
             "test_dataset_ratio": 0.0,
             "line_length_limit": 500,
             "lower_bound_instances": 500,
+            "overall_size_limit": 2 << 20,  # 2 MB
```

Comment:
why do you set exactly this number?

Reply:
Taken from jQuery. I guess everything which is bigger starts to cause trouble.